src/pkg/packaging/dir.proj

```diff
@@ -174,6 +174,8 @@
 
     <ItemGroup Condition="'$(BuildAllPackages)' == 'true'">
       <!-- The list of packages we are not servicing -->
+      <PackageProjects Include="$(ProjectDir)src\managed\Microsoft.DotNet.PlatformAbstractions\Microsoft.DotNet.PlatformAbstractions.csproj" />
```

Comment:
You also need to remove them from the other item group, but keep it around with just the comment.